python/tskit/provenance.schema.json

```diff
@@ -45,6 +45,28 @@
           "type": "object"
         }
       }
+    },
+    "resources": {
+      "description": "Resources used by this operation.",
+      "type": "object",
+      "properties": {
+        "elapsed_time": {
+          "description": "Wall clock time in used in seconds.",
+          "type": "number"
+        },
+        "user_time": {
+          "description": "User time used in seconds.",
+          "type": "number"
+        },
+        "sys_time": {
+          "description": "System time used in seconds.",
+          "type": "number"
+        },
+        "max_mem": {
```

Comment:
Let's make this ``max_memory`` here, I've commited this in sc2ts and the brevity doesn't really help.